.github/workflows/sync-wiki.yml

```
@@ -0,0 +1,187 @@
+        cp -r wiki/* wiki-backup/ 2>/dev/null || true
+        
+        # Copy documentation files from docs/wiki/ to wiki root
+        if [ -d "docs/wiki" ]; then
+          echo "📋 Copying documentation files..."
+          
+          # Copy all markdown files
+          find docs/wiki -name "*.md" -type f | while read file; do
+            # Convert file path to wiki format
+            relative_path=${file#docs/wiki/}
+            target_path="wiki/$relative_path"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)